src/DateTimePickerAndroid.android.js

```diff
@@ -118,7 +118,7 @@ function open(props: AndroidNativeProps) {
 
 function dismiss(mode: AndroidNativeProps['mode']): Promise<boolean> {
   // $FlowFixMe - `AbstractComponent` [1] is not an instance type.
-  return pickers[mode].dismiss();
+  return pickers[mode]?.dismiss();
```

Comment:
hello and thanks for opening the PR.
I'm sure this fixes the error of "Cannot read property 'dismiss' of undefined" but I'd like to know first why `pickers[mode]` is undefined. It shouldn't be, as far as I can tell.
In the issue that you linked there's a lot of comments, but no one was able to provide a reproduction of the issue.

Right now, I see this more as a workaround rather than a solution. I'd like to know better what the underlying problem is in order to be able to assess if this is the right fix.

Can you provide more insight into what's happening?
Thank you